includes/class-wc-google-gtag-js.php

```diff
@@ -304,12 +306,11 @@ function ' . self::tracker_var() . '(){dataLayer.push(arguments);}
 		} );
 
 		$track_404_enabled
-		</script>
 		";
 
-		$gtag_snippet = apply_filters( 'woocommerce_gtag_snippet' , $gtag_snippet );
+		wp_register_script( 'google-tag-manager', 'https://www.googletagmanager.com/gtag/js?id=' . esc_js( $gtag_id ), array(), WC_GOOGLE_ANALYTICS_INTEGRATION_VERSION, false );
+		wp_add_inline_script( 'google-tag-manager', apply_filters( 'woocommerce_gtag_snippet', $gtag_snippet ) );
```

Comment:
We also want to add a enqueue call here:

```php
wp_enqueue_script( 'google-tag-manager' );
```

The way this code is now it relies on the enqueue call from blocks `wp_enqueue_script( 'wc-blocks-google-analytics' );` because `google-tag-manager` is a dependent script. It's pretty hard to run a site without blocks but if there is ever any condition where blocks have been disabled, we still want our script to load.